docs/custom_conf.py

```diff
@@ -0,0 +1,216 @@
+import datetime
+
+# Custom configuration for the Sphinx documentation builder.
+# All configuration specific to your project should be done in this file.
```

Comment:
I'm not sure why we need both conf and custom_conf...

Is the idea to keep `conf.py` a copy of upstream and sync their changes?

Reply:
Per my understanding, I think the purpose is that you clone the starter pack, copy everything except the custom_conf, and only change one file that is custom_conf. It's probably useful when you need to pull in the latest changes from the starter pack (but based on my personal experiences, it probably doesn't work because the starter pack changes quickly).

Reply:
I think we should stick to the starter pack pattern. They really ought to be ensuring that `conf.py` can be replaced by just dropping in the new pack files without breaking `custom_conf.py`. It also helps make it clear what's "starter pack" and what's "pebble".